### PR TITLE
Additional coverage for Url annotation

### PR DESCRIPTION
**Pull request by benoitdion**
Assert Url works with the POST annotation and with a different host.

### CODE REVIEW

File: retrofit/src/test/java/retrofit/RequestBuilderTest.java [hunk 1]
```diff
@@ -910,6 +910,21 @@
     assertThat(request.body()).isNull();
   }
 
+  @Test public void getWithUrlAbsoluteDifferentHost() {
```
Comment (JakeWharton) [inline retrofit/src/test/java/retrofit/RequestBuilderTest.java]: This is already tested directly above. The base URL authority is 'example.com' and we switch it to 'example.org'. I realize they look very similar which can cause confusion. Maybe just change the above two tests to 'example2.com`?

Comment (benoitdion) [inline retrofit/src/test/java/retrofit/RequestBuilderTest.java]: That's fine, didn't see the .org difference. Any value in a test with the same authority?

Comment (JakeWharton) [inline retrofit/src/test/java/retrofit/RequestBuilderTest.java]: Sure. Can't hurt.

**Comment by benoitdion** [comment]
Made additional changes based on review comments

File: retrofit/src/test/java/retrofit/RequestBuilderTest.java [hunk 2]
```diff
@@ -903,10 +903,25 @@
       }
     }
 
-    Request request = buildRequest(Example.class, "http://example.org/foo/bar/");
+    Request request = buildRequest(Example.class, "https://example2.com/foo/bar/");
```
Comment (benoitdion) [inline retrofit/src/test/java/retrofit/RequestBuilderTest.java]: Note the `https` change

**Pull request merged**